### PR TITLE
Update AsyncSSH to cover version 1.18.0

### PR DESCRIPTION
**Pull request by ronf**

### CODE REVIEW

File: _impls/asyncssh.md [hunk 1]
```diff
@@ -63,15 +63,21 @@ protocols:
...
         - gss-gex-sha1-*                                # since 1.9.0
-        - gss-group1-sha1-*                             # since 1.9.0
-        - gss-group14-sha1-*                            # since 1.9.0
         - gss-group14-sha256-*                          # since 1.9.0
         - gss-group15-sha512-*                          # since 1.9.0
         - gss-group16-sha512-*                          # since 1.9.0
         - gss-group17-sha512-*                          # since 1.9.0
         - gss-group18-sha512-*                          # since 1.9.0
+        - gss-group14-sha1-*                            # since 1.9.0
+        - gss-group1-sha1-*                             # since 1.9.0
```
Comment (fingolfin) [inline _impls/asyncssh.md:80]: Why the move? Perhaps because they were introduced in an earlier version? (Then update the comment?)

**Review by fingolfin — APPROVED** [review]
Looks good, one minor question

File: _impls/asyncssh.md (hunk 1 above)
Comment (ronf) [reply _impls/asyncssh.md:80]: No - the version information is correct here. The move relates to these algorithms being downgraded from a security perspective due to weaknesses discovered in SHA1 and the small modulus size (1024 bits) associated with group1.

Preeviously, the DH version of group1-sha1 was a MUST, but https://tools.ietf.org/html/draft-ietf-curdle-ssh-kex-sha2-10 is recommending it be a SHOULD NOT for now eventually moving to a MUST NOT.  The DH version of group14-sha1 is moving from MUST to SHOULD, and will eventually become SHOULD NOT when SHA2 is more widely supported, and the GSS versions of both of these were similarly downgraded by the draft.

In line with the recommendations, I lowered the preference of these algorithms compared to other stronger algorithms. This was a level of detail I wasn't capturing in this file, though.

Comment (ronf) [reply _impls/asyncssh.md:80]: Is there any other info you need on this?

**Comment by ronf** [comment]
Thanks!

**Pull request merged**